docs/index.md

```diff
@@ -1 +1,100 @@
-# TODO
+## About
+
+**Blender Image Preview**, or BIP for short is a **[Blender](https://blender.org)**
+library by 3D Ninjas that allows addon developers to have extra functionality
+that comes built into Blender but takes quite some time to set up.
```

Comment:
Mmm.. its not really build-in functionality. Its rather that we provide extra functionality that takes a lot of work to implement on your own.

Reply:
Updated.